### PR TITLE
Fix of the issue of Kvasir build fail

### PR DESCRIPTION
**Pull request by sharkov63**
This is changes I made to complete building of Kvasir, see issue #56 .

### CODE REVIEW

File: valgrind/fjalar/dwarf.c [hunk 1]
```diff
@@ -58,7 +58,12 @@
 # define ngettext(Msgid1, Msgid2, n) \
   (n == 1 ? Msgid1 : Msgid2)
 //#include "libiberty.h"
+// 2022.07.06 uWX1176834 added startswith macro
```
Comment (mernst) [inline valgrind/fjalar/dwarf.c:61]: What is "uWX1176834"?  Is that you, or a marker for some commit, or ...?

**Review by mernst — COMMENTED** [review]
Thanks for these changes!

I wonder whether these changes are required because we haven't updated Fjalar/Kvasir from upstream recently, or whether they are a problem with the underlying Valgrind.

File: valgrind/fjalar/dwarf.c (hunk 1 above)
Comment (sharkov63) [reply valgrind/fjalar/dwarf.c:61]: That is me, it is my internal nickname in a company

Comment (mernst) [reply valgrind/fjalar/dwarf.c:61]: Thanks for the explanation.  Could you please either replace it, or augment it, by your GitHub ID and/or your name?  That will reduce confusion in the future.

Comment (sharkov63) [reply valgrind/fjalar/dwarf.c]: Done

**Comment by markro49** [comment]
Thank you for your suggested fix.  I made a few minor changes and resolved with commit: https://github.com/codespecs/fjalar/commit/353ee0086ae0ee78b90e5be32ad00f697c09a22a

**Pull request closed without merging**